src/test/java/io/appform/dropwizard/actors/actor/metadata/MessageDelayMetaGeneratorTest.java

```diff
@@ -0,0 +1,63 @@
+package io.appform.dropwizard.actors.actor.metadata;
+
+
+import static io.appform.dropwizard.actors.common.Constants.MESSAGE_PUBLISHED_TEXT;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.appform.dropwizard.actors.actor.MessageMetadata;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class MessageDelayMetaGeneratorTest {
```

Comment:
When moving the generator classes to `generators` move corresponding tests as well